web/src/components/gnd-feature-details/index.js

```diff
@@ -81,9 +82,11 @@ class GndFeatureDetails extends React.Component<Props> {
         </Typography>;
       });
       return <Card key={index} classes={{root: classes.card}}>
-        <CardHeader title={record.created.user.displayName} classes={{title: classes.title, root: classes.cardHeaderRoot}}/>
+        <CardHeader
+          title={record.created && record.created.user ? record.created.user.displayName : ""}
+          classes={{title: classes.title, root: classes.cardHeaderRoot}}/>
         <CardContent classes={{root: classes.cardContentRoot}}>
-          <p className={classes.date}>{record.created.clientTimestamp}</p>
+          <p className={classes.date}>{record.created ? record.created.clientTimestamp : ""}</p>
```

Comment:
Please assign this and display name to local var for readability. Could also use short circuit operators instead of ternary e.g.:
const created = record.created;
const creatorName = created && created.user && created.user.displayName;